### ISSUE TITLE
loss is 0

### ISSUE THREAD

**Issue by 3DMM-ICME2023** [samiksome/C3D-weights#2]
@samiksome  Thanks for your nice work!
Did you finetune your provided model on UCF101 and get the same accuracy as the C3D(82.3)?

**Comment by samiksome92** [comment]
No I did not finetune it on UCF101. I actually needed multi-label learning which is supported in v1.1 but wanted to use the pretrained v1.0 weights.

But regarding your problem, it's probably not the weights themselves. (Even if the weights were incorrect you should not get 0 loss and accuracy.)
